tests/integration/expected_test_results/search_v1/search-tags-categories-without-ingredients-from-palm-oil.json

```diff
@@ -74,7 +74,7 @@
             "2xxxxxxxxxxx"
          ],
          "complete" : 0,
-         "completeness" : "0.5",
+         "completeness" : 0.5,
```

Comment:
Do you understand why you have this flip-flap between string and raw numbers ?

Reply:
@alexgarel it's an issue with JSON:PP which select the type (string or number) based on what the last use of the scalar was. I tried to find out what exactly was using those scalars in a string context (like a debug dump function), but I failed. 

https://docs.mojolicious.org/JSON/PP#PERL---JSON